common/persistence/persistence-tests/metadataPersistenceV2Test.go

```diff
@@ -717,16 +717,10 @@ func (m *MetadataPersistenceSuiteV2) TestUpdateDomain() {
 	}
 
 	asyncWFCfg1 := types.AsyncWorkflowConfiguration{
-		QueueType: types.AsyncWorkflowQueueTypeKafka,
-		KafkaConfig: &types.AsyncWorkflowKafkaQueueConfiguration{
-			Topic: "topic1",
-		},
+		PredefinedQueueName: "queue1",
 	}
 	asyncWFCfg2 := types.AsyncWorkflowConfiguration{
-		QueueType: types.AsyncWorkflowQueueTypeKafka,
-		KafkaConfig: &types.AsyncWorkflowKafkaQueueConfiguration{
-			Topic: "topic2",
-		},
+		PredefinedQueueName: "queue2",
```

Comment:
Are we testing the dynamic config? I noticed that the type doesn't match our config.

Reply:
We will start the implementation for `PredefinedQueueName` part. Ignore the `KafkaConfig` field here. We need another round of update to replace that with a blob.